arches_lingo/src/arches_lingo/pages/SchemeList.vue

```diff
@@ -1 +1,61 @@
-<template>Schemes list placeholder</template>
+<script setup lang="ts">
+import { provide, ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import Panel from "primevue/panel";
+import ProgressSpinner from "primevue/progressspinner";
+
+import {
+    displayedRowKey,
+    selectedLanguageKey,
+} from "@/arches_references/constants.ts";
+import { ENGLISH } from "@/arches_lingo/constants.ts";
+import { bestLabel } from "@/arches_lingo/utils.ts";
+import ConceptTree from "@/arches_lingo/components/tree/ConceptTree.vue";
+
+import type { Ref } from "vue";
+import type { Language } from "@/arches/types";
+import type { Labellable } from "@/arches_lingo/types.ts";
+
+const { $gettext } = useGettext();
+
+// TODO: lift some of this state up?
+const displayedRow: Ref<Labellable | null> = ref(null);
+const setDisplayedRow = (val: Labellable | null) => {
+    displayedRow.value = val;
+    // TODO: routing
+};
```

Comment:
From speaking with @robgaston just now, FYI @chrabyrd that I think the next step will be to do what this TODO says and lift this state up somewhere so that the search component can set the displayed row (probably need to rename it displayed concept or scheme). displayed "item"? This was tricky in the controlled list manager, which had items, but we can probably use "item" here.